tokio/src/macros/select.rs

```diff
@@ -562,41 +563,41 @@ macro_rules! select {
     (@ { start=$start:expr; $($t:tt)* } else => $else:expr $(,)?) => {
         $crate::select!(@{ start=$start; $($t)*; $else })
     };
-    (@ { start=$start:expr; ( $($s:tt)* ) $($t:tt)* } $p:pat = $f:expr, if $c:expr => $h:block, $($r:tt)* ) => {
```

Comment:
Should I duplicate this whole block here to fix the recursion limit?

Reply:
I don't know how to fix it. You'll have to try it out.